tests/integration/test_fim/test_files/test_follow_symbolic_link/common.py

```diff
@@ -78,10 +78,3 @@ def extra_configuration_before_yield():
     create_file(SYMLINK, symlinkdir, 'symlink', target=os.path.join(testdir1, 'regular1'))
     # Symlink pointing to /testdir_target/
     create_file(SYMLINK, symlinkdir, 'symlink2', target=testdir_target)
-    # Set symlink_scan_interval to a given value
-    change_internal_options(param='syscheck.symlink_scan_interval', value=symlink_interval)
-
-
-def extra_configuration_after_yield():
-    """Set symlink_scan_interval to default value"""
-    change_internal_options(param='syscheck.symlink_scan_interval', value=600)
```

Comment:
`change_internal_options` is still being imported even though we don't use it. Furthermore, `change_internal_options` is no longer being used anywhere, we might want to delete it from fim.py too.